open-media-match/src/OpenMediaMatch/blueprints/matching.py

```diff
@@ -49,6 +50,21 @@ def lookup():
     return {"matches": [m.metadata for m in index.query(signal)]}
 
 
+def _validate_and_transform_signal_type(
+    signal_type_name: str, storage: IUnifiedStore
```

Comment:
nit: You could narrow this slightly since you are only getting the signal_type configs